backend/test/modules/player-profile/core/application/SendEmailAfterProfileWasCreated.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { testPlayerProfileModule } from './TestPlayerProfileModule';
+import { PlayerProfileWasCreated } from '../../../../../src/modules/player-profiles/core/domain/event/PlayerProfileWasCreated';
+import { CommandPublisher } from '../../../../../src/shared/core/application/command/CommandBus';
+import { SendEmailAfterPlayerProfileWasCreatedEventHandler } from '../../../../../src/modules/player-profiles/core/application/event/SendEmailAfterPlayerProfileWasCreatedEventHandler';
+
+describe('Send an email to player', () => {
+  it('When player profile is created, then execute command for email sending', () => {
+    //Given
+    const currentTime = new Date();
+    const playerProfileModuleCore = testPlayerProfileModule(currentTime);
+    const firstName = 'Johnny';
+    const emailAddress = 'bravo@gmail.com';
+
+    const playerProfileWasCreated = new PlayerProfileWasCreated({
+      occurredAt: currentTime,
+      playerId: 'PlayerId',
+      firstName: firstName,
+      lastName: 'Bravo',
+      emailAddress: emailAddress,
+      phoneNumber: '123456789',
+    });
+
+    const commandPublisher: CommandPublisher = {
+      execute: jest.fn(),
+    };
+    new SendEmailAfterPlayerProfileWasCreatedEventHandler(commandPublisher);
```

Comment:
Ten handler nie jest w żaden sposób spięty z testPlayerProfileModule.

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/37/commits/d7f7483ad72dba96f897978090063eb2b5a58d9b

czyli rozumiem, że wrzucam mu tego zamockowanego commandPublishera, ale jeszcze muszę mu jakoś przekazać eventHandlera z tymże oto zamockowanym commandPublisherem.... już tyle nad tym siedzę, że zupełnie przestałem ogarniać to tu się dzieje

Reply:
#update

 https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/37/commits/304f087d3b14429e27b88759ef43176b224a73e4
 takie coś zrobić jak jest w TODO